cmd/aport_l.go

```diff
@@ -32,9 +34,14 @@ var (
 
 `,
 		Run: func(cmd *cobra.Command, args []string) {
-			vcd.PrintAports(output, verbose)
+			manager := vcd.AppPortManager{
+				Client: vcdClient,
+			}
+			items := manager.List()
+			utils.Print(outputFormat, items, "name",
+				[]string{"NAME", "PROTOCOL", "PORTS"},
+				[]string{"name", "applicationPorts.0.protocol", "applicationPorts.0.destinationPorts.0"})
```

Comment:
will this fail if there is some `nil` in the json path? Perhaps if the `applicationPorts` is empty?